### PR TITLE
Add setup instructions to README

### PR DESCRIPTION
**Pull request by Kalikoze**
I added some setup instructions to clone down our new repo.  Is there anything that I am missing @kaylagordon!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,8 @@
 # Travis and Kayla's Super Awesome Repo
+
+## Setup Instructions
```
Comment (kaylagordon) [inline README.md:3]: I love that you used a second level heading here!

File: README.md [hunk 2]
```diff
@@ -1 +1,8 @@
...
+1. Clone down the repo
+```
```
Comment (kaylagordon) [inline README.md:5]: woah! nice code block!

File: README.md [hunk 3]
```diff
@@ -1 +1,8 @@
...
+git clone git@github.com:kaylagordon/git-practice-2201.git
+```
+2. Check out our amazing README
```
Comment (kaylagordon) [inline README.md]: Maybe we should add a step here for how to open the file.

**Review by kaylagordon — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (kaylagordon) [reply README.md]: @Kalikoze can you add a step here to show how to open the file?

File: README.md [hunk 4]
```diff
@@ -1 +1,11 @@
...
+
+## Setup Instructions
+1. Clone down the repo
+```
+git clone git@github.com:kaylagordon/git-practice-2201.git
+```
+2. Open README and review documentation
+```
+open README.md
+```
```
Comment (kaylagordon) [inline README.md:11]: Awesome!

**Review by kaylagordon — APPROVED** [review]
Looks great!

File: README.md [hunk 5]
```diff
@@ -1 +1,11 @@
 # Travis and Kayla's Super Awesome Repo
+
+## Setup Instructions
+1. Clone down the repo
+```
+git clone git@github.com:kaylagordon/git-practice-2201.git
+```
+2. Open README and review documentation
```
Comment (kaylagordon) [inline README.md:8]: something!

File: README.md [hunk 6]
```diff
@@ -1 +1,11 @@
 # Travis and Kayla's Super Awesome Repo
+
```
Comment (kaylagordon) [inline README.md:2]: something else!

**Review by kaylagordon — APPROVED** [review]
Let's merge!

File: README.md (hunk 3 above)
Comment (Kalikoze) [reply README.md]: Great suggestion Kayla, thank you!

**Pull request merged**